Found crash: class java.lang.AssertionError - 
Saved - /home/shuai/xlab/cfuzz/fuzz-hadoop/hadoop-common-project/hadoop-common/fuzz_output_round2/fuzz-results/org.apache.hadoop.security.TestDoAsEffectiveUser/testRealUserAuthorizationSuccessFuzz/failures/id_000000 random +crash
